lldb/test/API/functionalities/postmortem/elf-core/TestLinuxCore.py

```diff
@@ -760,6 +772,61 @@ def test_riscv64_regs(self):
 
         self.expect("register read --all")
```

Comment:
Re-generated executable and coredump for `*_fpr` test-cases using `main_fpr.c` instead of `main.c`, which has non-zero `fa5` FP-register during the crash. Adjusted test-cases accordingly in https://github.com/llvm/llvm-project/pull/104547/commits/e656524b63eed3a17e132e2280d44bf0e70e3c68